### DISCUSSION TITLE
Realease Every weakend with small feature changes.

### DISCUSSION THREAD

**Discussion by sd8917** [techy1999/BlogFrontend#61] [General]
# Release -003
- Include all ticket with **high priority** into this release  29 April 2024.

**Comment by techy1999** [comment]
Release is very near we had create label for priority ticket label [Weekend-Release](https://github.com/techy1999/BlogFrontend/issues?q=is%3Aissue+is%3Aopen+label%3AWeekend-Release)  .After completing all the ticket inside we will release ```0003``` version.

**Reply by techy1999** [reply]
done [release-003](https://github.com/techy1999/BlogFrontend/releases/tag/Realease-0003)

**Comment by techy1999** [comment]
# Release-004
- Include all ticket with high priority or [Weekend-Release](https://github.com/techy1999/BlogFrontend/issues?q=is%3Aissue+is%3Aopen+label%3AWeekend-Release) into this release on 5 May 2024.

**Reply by techy1999** [reply]
Done [Release note]( https://github.com/techy1999/BlogFrontend/releases/tag/Release-004)

**Comment by techy1999** [comment]
# Release-005 (Done)
- Include all ticket with high priority or [Weekend-Release](https://github.com/techy1999/BlogFrontend/issues?q=is%3Aissue+is%3Aopen+label%3AWeekend-Release) into this release on 12 May 2024.

**Comment by techy1999** [comment]
# Release-006 (Done)
- Include all ticket with high priority or [Weekend-Release](https://github.com/techy1999/BlogFrontend/issues?q=is%3Aissue+is%3Aopen+label%3AWeekend-Release) into this release on 19 May 2024.Release-006 (Done)

**Reply by techy1999** [reply]
Released [Release-006](https://github.com/techy1999/BlogFrontend/releases/tag/Release-0006) 
@Sky-De @anuescapist @sd8917 @techy1999 @dudekrystian @anuescapist

**Comment by techy1999** [comment]
# Release-007 (Upcoming)

- Include all ticket with high priority or [Weekend-Release](https://github.com/techy1999/BlogFrontend/issues?q=is%3Aissue+is%3Aopen+label%3AWeekend-Release) into this release on 26 May 2024.Release-007 (Upcoming)
